pkg/lint/lintersdb/manager.go

```diff
@@ -408,6 +408,11 @@ func (m Manager) GetAllSupportedLinterConfigs() []*linter.Config {
 			WithAlternativeNames("vet", "vetshadow").
 			WithURL("https://golang.org/cmd/vet/"),
 
+		linter.NewConfig(golinters.NewGrouper(m.cfg)).
+			WithSince("v1.22.0"). // TODO
```

Comment:
What do use here?

Reply:
The current version is `1.43.0` so this will be a part of the next minor release, it should be mentioned [under section 3](https://golangci-lint.run/contributing/new-linters/) in the docs.

```suggestion
			WithSince("v1.44.0").
```

Reply:
https://github.com/golangci/golangci-lint/blob/e3d0247ee24aed2fc231b21d3eaa04f0fd7eca47/pkg/lint/lintersdb/manager.go#L165-L166